src/backend/libs/efiling-account-client/src/main/java/ca/bc/gov/open/jag/efilingaccountclient/DemoAccountServiceImpl.java

```diff
@@ -12,21 +12,24 @@
 
     public static final UUID ACCOUNT_WITH_EFILING_ROLE = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fa");
     public static final UUID ACCOUNT_WITHOUT_EFILING_ROLE = UUID.fromString("77da92db-0791-491e-8c58-1a969e67d2fb");
+    public static final UUID ACCOUNT_DOES_NOT_EXISTS = UUID.fromString("88da92db-0791-491e-8c58-1a969e67d2fb");
 
     private Map<String, AccountDetails> csoAccounts = new HashMap<>();
 
     public DemoAccountServiceImpl() {
 
         AccountDetails accountWithEfilingRole = new AccountDetails(BigDecimal.TEN, BigDecimal.TEN, true, "", "", "", "");
         AccountDetails accountWithoutEfilingRole  = new AccountDetails(BigDecimal.TEN, BigDecimal.TEN, false, "", "", "", "");
+        AccountDetails accountWithoutCsoAccount  = new AccountDetails(BigDecimal.ZERO, BigDecimal.ZERO, false, "Bob", "Ross", "Rob", "bross@paintit.com");
```

Comment:
Is this code touched in production? Seems like test values?

Reply:
Kinda, the demo app for demoing cases without using the live soap servers.

Reply:
No, not in production.  Demo app is for showing what the efiling process looks like without hitting the live soap servers/object repositories etc.